packages/sqflite/analysis_options.yaml

```diff
@@ -0,0 +1,22 @@
+# This file was copied from Baseflow/flutter-geolocator/geolocator/analysis_options.yaml.
```

Comment:
I think you may need [this](https://github.com/tekartik/sqflite/blob/master/sqflite/analysis_options.yaml)